bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/adapters/TestColumnDescriptorAdapter.java

```diff
@@ -191,10 +191,8 @@ public void badExpressionLessThan(){
 
   @Test
   public void testBlankExpression(){
```

Comment:
testKeepAllVersions()  // to complete the conversion loop
1. blank gc expression, convert to HBase metadata
   verify that HBase max version is set to Integer.MAX_VALUE
2. convert the result from #1 to gc expression
  verify empty gc expression  
3. convert the result for #2 to HBase metadata again
  verify that the max version is still set to Integer.MAX_VALUE